### ISSUE TITLE
Not so clear what configuration we are solving for in the four bar linkage nsolve example

### ISSUE THREAD

**Issue by moorepants** [moorepants/learn-multibody-dynamics#185]
There are multiple solutions and I don't really specify why I chose -75 deg and 100 deg as the guesses.

**Comment by Peter230655** [comment]
I assume, you wanted a certain initial configuration, maybe you made a rough sketch of the configuration you wanted, and then estimated the angles $q_2, q_3$. Maybe worth mentioning, that the multiple solutions correspond to various realizeable configurations of the four-link system?

**Comment by moorepants** [comment]
Yes, it is missing a sketch that aligns with the configuration that I find. A student was confused about why I choose the guess I did and I realized that is not explained.
